tensorflow/java/src/gen/java/org/tensorflow/processor/operator/OperatorProcessor.java

```diff
@@ -0,0 +1,115 @@
+package org.tensorflow.processor.operator;
```

Comment:
Could we reduce one level of nesting and make this `org.tensorflow.processor`? Is there anything else in the `processor` package?

Reply:
This was mostly if anticipating processors for other annotations down the road, but also sgtm to reorganize later if needed. I've moved it up a level.